src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -468,7 +469,35 @@ void submitStreamReadOperation(final PathOperation streamOp, final String nodeAd
 	}
 
 	void submitStreamDeleteOperation(final PathOperation streamOp, final String nodeAddr) {
-		// TODO: Igor, issue SDP-49
+		try {
+			final String scopeName = DEFAULT_SCOPE; // TODO make this configurable
+			final String streamName = streamOp.dstPath();
```

Comment:
stream name is an item name for the `PathOperation` instances

Reply:
Fixed